src/tir/transforms/storage_access.cc

```diff
@@ -132,6 +132,7 @@ void StorageAccessVisitor::VisitStmt_(const AttrStmtNode* op) {
       StmtExprVisitor::VisitStmt_(op);
     }
     env_threads_.pop_back();
+  } else if (op->attr_key == attr::hand_threaded) {
   } else {
```

Comment:
How about this?
```
else if (op->attr_key != attr::hand_threaded) {
   StmtExprVisitor::VisitStmt_(op);
}
```
And better to leave a comment on what's up with `attr::hand_threaded`

Reply:
I'll add a comment, but the point of this is to skip thread planning for the enclosed block, i.e., not to visit the statements in this block. I do that because 1) it's already threaded and 2) the thread planning pass can't handle the per-thread read/write/sync I'm doing.